src/views/view.cpp

```diff
@@ -19,7 +19,7 @@ namespace views {
     void View::render(bool force) {
         if (force)
         {
-            clear();
+            erase();
```

Comment:
ho provato a vedere se erase era più veloce di clear, visto che chiama un metodo in meno, ma da quel che ho visto è praticamente equivalente.